### ISSUE TITLE
Two camera

### ISSUE THREAD

**Issue by WHGang** [atenpas/gpd#83]
Hello, when I read your paper, I found that two cameras can be used. Can I use two cameras directly on the code you Shared? Do I need to make any changes?Thank you!

**Comment by atenpas** [comment]
Yes, you can. With ROS, you should use [this message](https://github.com/atenpas/gpd_ros/blob/mast/msg/CloudSources.msg). Without ROS, you could provide a PCD file that has been generated by concatenating two point clouds.

**Comment by FrankLin9981** [comment]
Excuse me. I have the same question about how to use two cameras with ROS.
But the link you provided is broken. Could you provide some details about how to use two cameras?
Thanks.

**Comment by atenpas** [comment]
Yes, have a look at this ROS message: [CloudSources.msg](https://github.com/atenpas/gpd_ros/blob/master/msg/CloudSources.msg).

**Comment by FrankLin9981** [comment]
Thanks for your reply!
So, it looks like that I should create this type of message and publish it by myself.
Then modify the cloud_type parameter in launch file?
Do I misunderstand the mechanism?
Or there has been already wrapped by your package?

**Comment by atenpas** [comment]
Sorry, I pointed you to the wrong message. The grasp detection node can be used with [CloudIndexed](https://github.com/atenpas/gpd_ros/blob/master/msg/CloudIndexed.msg) or [CloudSamples](https://github.com/atenpas/gpd_ros/blob/master/msg/CloudSamples.msg) messages. Both of these messages have a [CloudSources](https://github.com/atenpas/gpd_ros/blob/master/msg/CloudSources.msg) member.

**Comment by FrankLin9981** [comment]
So, I should generate [CloudIndexed](https://github.com/atenpas/gpd_ros/blob/master/msg/CloudIndexed.msg) or [CloudSamples](https://github.com/atenpas/gpd_ros/blob/master/msg/CloudSamples.msg) by myself?
Or there has been already wrapped by your package?
Thanks.
